experiments/speedrun/llama_75m_adamax/llama_75m_adamax.py

```diff
@@ -62,6 +62,12 @@ def _optimizer(learning_rate):
 logger = logging.getLogger("ray")
 
 speedrun_config = SpeedrunConfig(
+    author=Author(
+        name="Nikil Ravi",
+        affiliation="Stanford University",
+        url="https://www.linkedin.com/in/nikilravi/",
+    ),
+    description="75M param model with Adamax optimizer",
```

Comment:
```suggestion
    description="75M parameter model with Adamax optimizer",
```
Is it possible to update the description without doing a rerun (e.g., by using cached results)?

Reply:
Yes, we can use override_output_path for the description and also anything that isn't run-related. `default_speedrun()` has an `override_output_path` argument already and uses the path's wandb link to reformat the description/results.